demo/minimalist.js

```diff
@@ -0,0 +1,93 @@
+const {
+  xml,
+  Component,
+  whenReady,
+  onWillStart,
+  onMounted,
+  useState,
+  onWillUnmount,
+  useExternalListener,
+  onError,
```

Comment:
not sure if we want all the possible helpers from owl or just the ones we need here?  if the former,  I'd at least add 'useEffect'

Reply:
I just forgot to clean everything up, the goal was to be minimalist :)